assets/benchmarks/shaders/Benchmark_Texture.hlsl

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "VsOutput.hlsli"
+
+Texture2D Tex0 : register(t0);
+
+float4 psmain(VSOutputPos input) : SV_TARGET
+{
+    return Tex0.Load(uint3(input.position.x, input.position.y, /* mipmap */ 0));
```

Comment:
sorry i couldn't find the vs Benchmark_VsSimpleQuads.vs, but how do you calculate the texture coord?

Reply:
I did not add texture coord to the vertex buffer, I am just using the position data here as coordinates for which pixel of the texture to load. I tested this manually with the `--resolution NxM` flag for different resolutions and it seems to be behaving as expected?

Reply:
`Benchmark_VsSimpleQuads.vs` file is added in this PR, just not with the most recent commit

Reply:
with the current position, the part covers the full screen will have a range of [-1, 1] whereas if we want to have 1to1 mapping between the texel and pixel, the range should be [0,1]?

Reply:
I see your point but I think that something else must be going on since this code produces a result that looks correct and doesn't look like I am accessing the texture out of bounds. I'm not an expert but I think it might have something to do with the HLSL POSITION->SV_POSITION change that the vertex shader is doing?

Reply:
As in I think `input.position` here is in range `0,0` to `window dimensions`, and it's going to access texture pixel with pixel coordinates (range `0,0` to texture size `3840-1, 2160-1`)

Reply:
if you could share a renderdoc capture, i can take a look

Reply:
I see, because we use SV_POSITION as pixel input which gives us the pixel location in the range of [0-screen width/height], and we use .Load which takes the pixel location instead of the texture coord to load the image. it works well. thanks!